upgrade/UPGRADE-v9.1.0-dev.md

```diff
@@ -31,3 +31,6 @@ There you can find links to upgrade notes for other versions too.
 - add phpstan-symfony extension ([#1961](https://github.com/shopsys/shopsys/pull/1961))
     - see #project-base-diff to update your project
 
+- introduced sniff for strict comparasion ([#1658](https://github.com/shopsys/shopsys/pull/1658))
+    - this change is optional but will be required in next major version
```

Comment:
Please remove this line as it is not optional, cause it will be required ;)

Reply:
It will be required in next major release, not before ;)

Reply:
My point still stands. It will be required. Upgrade notes are required when upgrading to next major, not to patch or minor version.